src/pybind/mgr/status/module.py

```diff
@@ -279,7 +279,7 @@ def handle_fs_status(self,
             return HandleCommandResult(stdout=output)
 
     @CLIReadCommand("osd status")
-    def handle_osd_status(self, bucket: Optional[str] = None) -> Tuple[int, str, str]:
+    def handle_osd_status(self, bucket: Optional[str] = None, format: str = 'plain') -> Tuple[int, str, str]:
```

Comment:
Please have a look at 

https://github.com/ceph/ceph/blob/7af7255fef5d054e94fa308eff4d884967cab798/src/pybind/mgr/orchestrator/module.py#L372

to properly propagate possible values to the CLI tool

Reply:
Yes, already change to this format. when use xml, it raise a ValueError: 'xml' is not a valid Format

Reply:
yes, cephadm is also affected by this. But as XML is nowadays less common, I'm ok with ignoring this for now.